### PR TITLE
Springster Redesign Phase 3

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: gem/styles/gem-springster/modules/tags/_m-tag-cloud.scss [hunk 1]
```diff
@@ -64,17 +64,17 @@
 }
 
 // //Tag Cloud for Article Page, dont see the need of this?
```
Comment (Mitso) [inline gem/styles/gem-springster/modules/tags/_m-tag-cloud.scss]: @projectmushroom this style tag is for article tag variation e.g. 
screenshot 2017-05-08 13 08 56

**Comment by Mitso** [comment]
@projectmushroom you have updated the following button
 `.call-to-action__nav-item-text--right` and `.call-to-action__nav-item-text--left`
Please note the back [left] button looks like this for me 
screenshot 2017-05-08 13 40 24

**Comment by projectmushroom** [comment]
@Mitso I'll sort out the button arrows, I like the  articles tags, nice 👍

File: gem/settings/base.py [hunk 2]
```diff
@@ -127,7 +127,7 @@
 
 # We have multiple layouts: use `base`, `malawi` or `springster`
 # to switch between them.
-SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'base')
+SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'springster')
```
Comment (Mitso) [inline gem/settings/base.py]: We need to update this

**Comment by Mitso** [comment]
💯 🍰

**Pull request merged**